app/assets/stylesheets/pages/_admin_login.scss

```diff
@@ -0,0 +1,16 @@
+.main_login {
+  padding-top: 5rem;
+
+  h1 {
+    color: $wheat_button;
+    margin-top: 0;
+  }
+
+  form {
+    label{
```

Comment:
Please add space between label and your brace

Reply:
Done